mission_1.css

```
@@ -0,0 +1,262 @@
+  margin:0;
+  padding:0;
+}
+header{
+  width: 1920px;
+  height: 70px;
+  padding: 10px 200px 9px 200px;
+  gap: 0px;
+  border: 0px 0px 1px 0px;
+  opacity: 0px;
+  background: #FFFFFF;
+  border-bottom: 1px solid #DFDFDF;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.logo{
+  margin: auto;
+  cursor: pointer;
+}
+.login{
+  width: Fixed (128px)px;
+  height: Fixed (48px)px;
+  top: 127px;
+  left: 538px;
+  padding: 12px 20px 12px 20px;
+  gap: 10px;
+  border-radius: 8px;
+  opacity: 0px;
+  background: #3692FF;
+  
+  margin:auto;
+  cursor: pointer;
+}
+.login_click{
+  color: #FFFFFF;
+}
+main{
+  
+  background: #CFE5FF;
+}
+.main_section{
+  width: 1920px;
+  height: 540px;
+  gap: 0px;
+  opacity: 0px;
+  background: #CFE5FF;
+  display:flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+.left{
+  display: flex;
+  flex-direction: column;
+  top: 170px;
+  left: 360px;
+  gap: 32px;
+  position: absolute;
+}
+h2{
+  color: #374151;
+  font-family: Pretendard;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  text-align: left;
+}
+.gotosee{
+  width: Hug (357px)px;
+  height: Hug (56px)px;
+  top: 127px;
+  left: 103px;
+  padding: 16px 124px 16px 124px;
+  gap: 10px;
+  border-radius: 40px;
+  opacity: 0px;
+  background: #3692FF;
+  
+  cursor: pointer;
+}
+.gotosee_click{
+  color:#ffffff;
+}
+.panda{
+  left: 701px;
+  bottom:0px;
+  padding-left: 30px;
+  position: absolute;
+}
+.container_box{
+  width: 1920px;
+  height:2160px;
+  text-align: center;
+}
+.container{
+  width: 1200px;
+  height: 2160px;
+  top: 610px;
+  left: 360px;
+  gap: 0px;
+  border: 1px;
+  display:grid;
+  grid-template-rows: repeat(3,1fr);
+  margin:auto;
+}
+.section{
+  display: flex;
+  align-items: center;
+  gap:64px;
+}
+.description{
+  display:flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap:12px;
+}
+.section_image{
+  width: 588px;
+  height: 444px;
+}
+.article1{
+  color: #3692FF;
+  font-family: Pretendard;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 25.2px;
+  text-align: left;
+}
+.article2{
+  font-family: Pretendard;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  letter-spacing: 0.02em;
+  text-align: left;
+  color: #374151;
+}
+.article3{
+  font-family: Pretendard;
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 28.8px;
+  letter-spacing: 0.08em;
+  text-align: left;
+  color: #374151;
+  margin-top:24px;
+}
+.section_reverse{
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: center;
+  gap:64px;
+}
+.second_description{
+  display:flex;
+  flex-direction: column;
+  align-items: flex-end;
+  gap:12px;
+}
+.article2_reverse{
+  font-family: Pretendard;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 56px;
+  letter-spacing: 0.02em;
+  text-align: right;
+  color: #374151;
+}
+.article3_reverse{
+  font-family: Pretendard;
```

Pretendard 같은 웹 폰트를 사용한다면 폴백용 폰트를 추가해주는 것이 좋습니다. 가장 기본적인 폰트가 sans-serif 체 이기 때문에 아래와 같이 Pretendard를 로드하지 못했을 때를 대비해서 작성해주시는 것 이 좋아요.
```suggestion
  font-family: Pretendard , sans-serif;
```